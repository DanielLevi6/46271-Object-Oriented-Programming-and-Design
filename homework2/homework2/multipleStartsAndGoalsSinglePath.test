# multipleStartsAndGoalsSinglePath test file

# Create node1 graph
CreateGraph myGraph

# Add nodes
CreateNode node1 1
CreateNode node2 1
CreateNode node3 4
CreateNode node4 2
CreateNode node5 2

AddNode myGraph node1
AddNode myGraph node2
AddNode myGraph node3
AddNode myGraph node4
AddNode myGraph node5

# Add an edge
AddEdge myGraph node1 node3
AddEdge myGraph node2 node3
AddEdge myGraph node2 node4
AddEdge myGraph node3 node4
AddEdge myGraph node4 node5

# Find node1 path
FindPath myGraph node1 node4 -> node2 node5 