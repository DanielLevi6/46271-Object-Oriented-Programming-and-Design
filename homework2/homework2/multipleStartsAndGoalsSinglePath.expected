# multipleStartsAndGoalsSinglePath test file

# Create node1 graph
created graph myGraph

# Add nodes
created node node1 with cost 1
created node node2 with cost 1
created node node3 with cost 4
created node node4 with cost 2
created node node5 with cost 2

added node node1 to myGraph
added node node2 to myGraph
added node node3 to myGraph
added node node4 to myGraph
added node node5 to myGraph

# Add an edge
added edge from node1 to node3 in myGraph
added edge from node2 to node3 in myGraph
added edge from node2 to node4 in myGraph
added edge from node3 to node4 in myGraph
added edge from node4 to node5 in myGraph

# Find node1 path
shortest path in myGraph: node4 node5
